cram_common/cram_common_failures/tests/different-location-solution-test.lisp

```diff
@@ -0,0 +1,100 @@
+;;;
+;;; Copyright (c) 2019, Amar Fayaz <amar@uni-bremen.de>
+;;; All rights reserved.
+;;;
+;;; Redistribution and use in source and binary forms, with or without
+;;; modification, are permitted provided that the following conditions are met:
+;;;
+;;;     * Redistributions of source code must retain the above copyright
+;;;       notice, this list of conditions and the following disclaimer.
+;;;     * Redistributions in binary form must reproduce the above copyright
+;;;       notice, this list of conditions and the following disclaimer in the
+;;;       documentation and/or other materials provided with the distribution.
+;;;     * Neither the name of the Institute for Artificial Intelligence/
+;;;       Universitaet Bremen nor the names of its contributors may be used to
+;;;       endorse or promote products derived from this software without
+;;;       specific prior written permission.
+;;;
+;;; THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS "AS IS"
+;;; AND ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT LIMITED TO, THE
+;;; IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS FOR A PARTICULAR PURPOSE
+;;; ARE DISCLAIMED. IN NO EVENT SHALL THE COPYRIGHT OWNER OR CONTRIBUTORS BE
+;;; LIABLE FOR ANY DIRECT, INDIRECT, INCIDENTAL, SPECIAL, EXEMPLARY, OR
+;;; CONSEQUENTIAL DAMAGES (INCLUDING, BUT NOT LIMITED TO, PROCUREMENT OF
+;;; SUBSTITUTE GOODS OR SERVICES; LOSS OF USE, DATA, OR PROFITS; OR BUSINESS
+;;; INTERRUPTION) HOWEVER CAUSED AND ON ANY THEORY OF LIABILITY, WHETHER IN
+;;; CONTRACT, STRICT LIABILITY, OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE)
+;;; ARISING IN ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE
+;;; POSSIBILITY OF SUCH DAMAGE.
+
+(in-package :cram-common-failures-tests)
+
+(defparameter *target-location* (cl-tf:make-pose-stamped
```

Comment:
Please don't use ``cl-tf``. It is shorter than ``cl-transforms`` and much shorter than ``cl-transforms-stamped`` but ``make-pose-stamped`` is from ``cl-transfroms-stamped`` package, not ``cl-tf``, and ``make-3d-vector`` is from ``cl-transforms``. Using ``cl-tf`` introduces a dependency on ``cl-tf``, which implements TF tree caching, and we don't need that here. Using ``cl-tf`` as a namespace will be removed in version 1.0 of CRAM, such that using it for symbols from other packages won't be possible anymore. And, yes, I understand that the autocompletion suggests ``cl-tf`` for these things but I don't want to introduce the dependency on ``cl-tf`` where it is not necessary.